lms/templates/main.html

```diff
@@ -184,7 +184,7 @@
         <%include file="/preview_menu.html" />
     % endif
 
-    % if course and course.self_paced and tab and relative_dates_is_enabled:
+    % if course and course.self_paced and relative_dates_is_enabled and reset_deadlines_banner_can_display:
```

Comment:
I guess my earlier comment could just be rephrased as a instead of passing in this `relative_dates_is_enabled` parameter here, you could wrap it into the `can_display` variable. Likely a matter of taste though